src/Build.OM.UnitTests/Definition/ProjectItem_Tests.cs

```diff
@@ -831,17 +831,27 @@ private static void ProjectGetterResultsInDriveEnumerationWarning(string unevalu
                     Helpers.ResetStateForDriveEnumeratingWildcardTests(env, "0");
 
                     // Setup
+                    string content = @"
+                    <Project>
+                        <ItemGroup>
+                            <i Include='i1'/>
+                        </ItemGroup>
+                    </Project>
+                    ";
+
+                    TransientTestProjectWithFiles testProject = env.CreateTestProjectWithFiles("build.proj", content);
                     ProjectCollection projectCollection = new ProjectCollection();
                     MockLogger collectionLogger = new MockLogger();
                     projectCollection.RegisterLogger(collectionLogger);
-                    Project project = new Project(projectCollection);
+                    Project project = new Project(testProject.ProjectFile, null, null, projectCollection);
```

Comment:
What is the behavior on an in-memory project? Is it still reasonable? Should this be a new test instead of a change to the existing one?

Reply:
There didn't seem to be any sort of visible effect on the contents of the in-memory project, so I don't think this test is actually reasonable. Reverting to original